t/ust.t

```diff
@@ -0,0 +1,275 @@
+                        'vsize'    => 256,
+                        'size'     => 256,
+                        'version'  => 2,
+                        'locktime' => 1666275,
+                        'hash'     => '19e5bda81d0a588cbf46072fb6da01685a51082d4b94428e09f491ad8a111bc7'
+                    },
+                    {
+                        'txid'     => 'abd322cee71dbd9adbd8b200df4c69a7870251f73ec6e04a3eb5e9aa2b065bdf',
+                        'locktime' => 0,
+                        'hex' =>
+                            '020000000203fdc4ea0591de6148363a8db7da8fb398bd3b1a84eb7d040aae787eb8acc690000000006a4730440220054a95f74fb2048a89e3e2a6961048a2a5539367f8f902083a38ac9b2afd458202206c9be9405bd12dfa0aa40c8ae299078bfd0730ea2d88f96850b169479cd9184f0121021f2a6cb7937966f4c2030e3817382ef0d2ab50482672a996a170528cda375bfcffffffff233d9e7e960adadc83c51413a6c77b705fc166804b9d23261cb49ccdbab68ba6020000006a473044022038ab0ddaaf35c989d9e7cf79a6538b54f96b4c1979960f060834ce98558a7ddc0220076632d3b468f7c24d6da0b8c7133bfdfb78ceac2340ac154d1bf304234d37b10121021f2a6cb7937966f4c2030e3817382ef0d2ab50482672a996a170528cda375bfcffffffff02b6140000000000001976a9140df9a2a1e8dd1ba5cb02bc2b3b67c8b7ba2f805088ac0000000000000000166a146f6d6e690000000000000001000000000098968000000000',
+                        'vsize'   => 369,
+                        'version' => 2,
+                        'size'    => 369,
+                        'hash'    => 'abd322cee71dbd9adbd8b200df4c69a7870251f73ec6e04a3eb5e9aa2b065bdf'
+                    }
+
+                ],
+                'bits' => '1d00ffff',
+                'time' => 1582166123
+            };
+            return Future->done($getblock);
+        },
+        list_by_addresses => sub {
+            my ($self, $address) = @_;
+            my $result;
+            if ($address eq 'mgVxUb5mYJkfoo4w2JBVBcWaP5bqtLroTr') {
+                $result = [{
+                        txids   => ['1b007afef8d271861dccd5b6bfe2f7e45a481c9a2f39b9f063e43e7d26375ec7'],
+                        address => 'mgVxUb5mYJkfoo4w2JBVBcWaP5bqtLroTr'
+                    }];
+            }
+
+            else { $result = []; }
+
+            return Future->done($result);
+        },
+        get_transaction => sub {
+            my ($self, @params) = @_;
+            my $omni_gettransaction = {
+                blocktime        => 1582166123,
+                confirmations    => 1,
+                blockhash        => '00000000000b031f58f9761fd1e0219ef5178481f274b4f54d30c5142b97571f',
```

Comment:
can we make this one, and 'from' 'to' address as variables ? The const string is so long and error-prone.

Reply:
And  variable will make code more clear and understandable.

Reply:
Updated the strings with variable . 👍